core/include/moveit/task_constructor/stages/pick.h

```diff
@@ -32,19 +32,27 @@
  *  POSSIBILITY OF SUCH DAMAGE.
  *********************************************************************/
 
-/* Authors: Robert Haschke */
+/* Authors: Robert Haschke, Artur Karoly */
 
 #pragma once
 
+#include <pluginlib/class_loader.h>
 #include <moveit/macros/class_forward.h>
 #include <moveit/task_constructor/container.h>
+#include <moveit/task_constructor/stages/grasp_provider.h>
+#include <moveit/task_constructor/stages/place_provider.h>
+#include <moveit/task_constructor/stages/modify_planning_scene.h>
 #include <geometry_msgs/TwistStamped.h>
+#include <geometry_msgs/Vector3Stamped.h>
+#include <geometry_msgs/PoseStamped.h>
+#include <Eigen/Geometry>
 
 namespace moveit {
 namespace task_constructor {
 
 namespace solvers {
 MOVEIT_CLASS_FORWARD(CartesianPath)
+MOVEIT_CLASS_FORWARD(PipelinePlanner)
 }
 
 namespace stages {
```

Comment:
I would add that the `Pick` and `Place` stages inherit from this `PickPlaceBase` class, and that it's not meant to be used directly.